.github/workflows/run-enterprise-action.yml

```diff
@@ -15,9 +15,15 @@ on:
       override_load_generators:
         type: string
         required: false
+      fail_action_on_run_failure:
+        type: boolean
+        default: 'true'
+      wait_for_run_end:
+        type: boolean
+        default: 'true'
```

Comment:
Feeling: it seem weird to me to have two attributes here.
If `wait_for_run_end` is false and `fail_action_on_run_failure` is true, what happend?

Reply:
Basically, nothing happens in that case. ^^ (I mean, it's just fire & forget anyway.)
I don't think there is really a good solution for that, since the action just has a list of text parameters (not structured parameters). 🤷 

But I'm making a note to try and come up with a good explanation to include in the inputs descriptions, at least.

Reply:
perhaps something like:
```yaml
action_result:
  type: choice
  options:
    - do_not_wait_for_end
    - fail_action_on_run_failure
    - wait_for_end
  default: wait_for_end
```

Reply:
Oh, you mean for the workflow? This workflow is just for manual testing, I'd rather it directly expose the action's input (so we _can_ test the weird combinations if we want).